charts/zookeeper/templates/zookeeper.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: {{ template "zookeeper.fullname" . }}
 spec:
   replicas: 3
+  image:
```

Comment:
What is this for?

Reply:
This field is needed to use the image mentioned in the values.yaml file